content/id/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Kelola Sertifikat TLS Pada Klaster
+content_template: templates/task
+reviewers:
+- girikuncoro
```

Comment:
this is not needed, we'll use `reviewers` when we go through the existing/translated docs to maintain quality